src/eventTile.ts

```diff
@@ -69,19 +72,34 @@ const handleEventCreationClick = (event: MouseEvent): void => {
     }
 };
 
-const placeNewEventTile = (currentEventTile: HTMLElement, eventData: EventData): void => {
-    const eventTileTitle: HTMLElement = currentEventTile.querySelector(".event-tile-title");
+const placeNewEventTile = (currentEventTile: HTMLElement, eventData: EventData) => {
+    const eventTileTitle: HTMLElement | null = currentEventTile.querySelector(".event-tile-title");
 
-    eventTileTitle.innerText = eventData.title;
+    if (eventTileTitle) eventTileTitle.innerText = eventData.title;
```

Comment:
We are avoiding if statement without the `{}`:
```suggestion
    if (eventTileTitle) {
      eventTileTitle.innerText = eventData.title
    };
```